VTEX - Audience API.json

```diff
@@ -0,0 +1,324 @@
+                        }
+                    }
+                }
+            }
+        },
+        "/api/price-table-mapper/pvt/mapping/{audienceId}": {
+            "get": {
+                "tags": [
+                    "Price Table Mapper"
+                ],
+                "summary": "Get price table mapping",
+                "description": "Fetches the price table mapping for a specific audience.\r\n\r\n## Permissions\r\nThis endpoint requires any one of the [predefined roles](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy).",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "$ref": "#/components/parameters/AudienceId"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "description": "Array of price table Ids.",
+                                    "items": {
+                                        "type": "string",
+                                        "description": "Price table ID."
+                                    }
+                                },
+                                "example": [
+                                    "priceTable01"
+                                ]
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/responseError"
+                                },
+                                "example": {
+                                    "error": "Mapping not found for the provided audienceId."
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "Price Table Mapper"
+                ],
+                "summary": "Set price table mapping",
+                "description": "Sets one or more price table mappings for the specified audience. Each price table ID in the array defines a distinct price table that applies to the audience.\r\n\r\n## Permissions\r\nThis endpoint requires any one of the [predefined roles](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy).",
```

Comment:
As I said, "no resources are necessary, in which case you would go with one of [these templates](https://www.notion.so/vtexhandbook/Endpoint-permissions-e359ebdc14ac40988095c9936f9db6d5?pvs=4#ae3674ee7e974acfa927976d3d522e08), depending on authentication."

This is the correct text in this case, becasuse you don't need any role:

`## Permissions\r\n\r\nThis endpoint does not require [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).`